examples/example3/example3_withmanufactured.ipynb

```diff
@@ -0,0 +1,679 @@
+    "    elif np.isclose(z, zSize):\n",
+    "        facet_markers[f] = 15\n",
+    "# domain, facet_markers, cell_markers = mesh_tools.create_spheres(curRadius, 0, hEdge=0.05, hInnerEdge=0.05)\n",
+    "# visualization.plot_dolfin_mesh(domain, cell_markers, facet_markers)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "3967a934",
+   "metadata": {},
+   "source": [
+    "In order to load this into a `ParentMesh` object, we need to first save it and then load it in using the `smart.mesh.ParentMesh` function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "94a798bd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Write mesh and meshfunctions to file\n",
+    "mesh_folder = pathlib.Path(f\"mesh_{curRadius:03f}/\")\n",
+    "mesh_folder.mkdir(exist_ok=True)\n",
+    "mesh_file = mesh_folder / \"DemoSphere.h5\"\n",
+    "mesh_tools.write_mesh(domain, facet_markers, cell_markers, filename=mesh_file)\n",
+    "# # Define parent mesh\n",
+    "parent_mesh = mesh.ParentMesh(\n",
+    "    str(mesh_file),\n",
+    "    mesh_filetype=\"hdf5\",\n",
+    "    name=\"parent_mesh\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "309e5e8a",
+   "metadata": {},
+   "source": [
+    "Define manufactured solution and associated reactions. To not make the choice of manufactured solution completely arbitrary, we have chosen it as a multiplication of three solutions to the PDE in 1D, then multiplied by some time-dependence governed by the time-scale of protein dephosphorylation in the cytosol."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6ce6af50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = sym.Symbol('x')\n",
+    "y = sym.Symbol('y')\n",
+    "z = sym.Symbol('z')\n",
+    "t = sym.Symbol('t')\n",
+    "k_kin = kkin.value*VolSA.value\n",
+    "k_p = kp.value\n",
+    "cT = Atot.value\n",
+    "D = Aphos.D\n",
+    "xFactor = xSize / np.sqrt(3*D/k_p)\n",
+    "Ax = (k_kin*xSize/(2*D))*np.exp(xFactor) / ((xFactor/2)*(np.exp(xFactor)-1) + (k_kin*xSize/(2*D))*(1+np.exp(xFactor)))\n",
+    "Bx = (k_kin*xSize/(2*D)) / ((xFactor/2)*(np.exp(xFactor)-1) + (k_kin*xSize/(2*D))*(1+np.exp(xFactor)))\n",
+    "yFactor = ySize / np.sqrt(3*D/k_p)\n",
+    "Ay = (k_kin*ySize/(2*D))*np.exp(yFactor) / ((yFactor/2)*(np.exp(yFactor)-1) + (k_kin*ySize/(2*D))*(1+np.exp(yFactor)))\n",
+    "By = (k_kin*ySize/(2*D)) / ((yFactor/2)*(np.exp(yFactor)-1) + (k_kin*ySize/(2*D))*(1+np.exp(yFactor)))\n",
+    "zFactor = zSize / np.sqrt(3*D/k_p)\n",
+    "Az = (k_kin*zSize/(2*D))*np.exp(zFactor) / ((zFactor/2)*(np.exp(zFactor)-1) + (k_kin*zSize/(2*D))*(1+np.exp(zFactor)))\n",
+    "Bz = (k_kin*zSize/(2*D)) / ((zFactor/2)*(np.exp(zFactor)-1) + (k_kin*zSize/(2*D))*(1+np.exp(zFactor)))\n",
+    "expFactor = np.sqrt(k_p/(3*D))\n",
+    "Asol = cT * ((Ax*sym.exp(-expFactor*x) + Bx*sym.exp(expFactor*x)) * \n",
```

Comment:
There is a mixture of `numpy.exp` and `sympy.exp` here, which might confuse the reader, is there any particular reason we cannot use just `sympy` here?

Reply:
Good catch, should just be fine as just `sympy.exp` I think.